src/EventEdition/GuestGrid.vala

```diff
@@ -106,8 +108,15 @@ public class Maya.View.EventEdition.GuestGrid : Gtk.Grid {
                         individual = map_iterator.get_value ();
                         if (individual != null) {
                             try {
```

Comment:
You don't need this `try{} catch(){}` clause anymore :slightly_smiling_face:

Reply:
True. 😅 Sorry, I'm currently on mobile so I've applied your suggestions through web and I didn't notice. I'll update the PR as soon as i come back home. 👍🏻

Reply:
Done.